types/types.go

```diff
@@ -377,8 +377,8 @@ type Resource struct {
 
 type DeviceRequest struct {
 	Capabilities []string `mapstructure:"capabilities" yaml:"capabilities,omitempty" json:"capabilities,omitempty"`
-	Diver        string   `mapstructure:"driver" yaml:"driver,omitempty" json:"driver,omitempty"`
-	Count        int      `mapstructure:"count" yaml:"count,omitempty" json:"count,omitempty"`
+	Driver       string   `mapstructure:"driver" yaml:"driver,omitempty" json:"driver,omitempty"`
+	Count        int64    `mapstructure:"count" yaml:"count,omitempty" json:"count,omitempty"`
```

Comment:
While fixing this typo, please note `count` can also be set to `all` and should be of type string